test/login-web-ui-test.js

```diff
@@ -52,34 +52,33 @@ suite.addBatch({
             },
             "and we visit the login URL": {
                 topic: function() {
-                    var browser;
+                    var cb = this.callback;
                     browser = new Browser({runScripts: true});
```

Comment:
True... 
I think this is a typical jshint case... 
Reminder to myself to start using jshint :smile: